features/attached_enable.feature

```diff
@@ -1,31 +1,31 @@
 @uses.config.contract_token
 Feature: Enable command behaviour when attached to an UA subscription
 
-    @series.trusty
-    Scenario: Attached enable Livepatch service in a trusty lxd container
-        Given a `trusty` lxd container with ubuntu-advantage-tools installed
+    Background: Attached enable a service in a trusty lxd container
+        Given a series lxd container with ubuntu-advantage-tools installed
+        |  series   |
+        |  trusty   |
+        |  focal    |
         When I attach contract_token with sudo
         And I run `ua enable livepatch` as non-root
```

Comment:
A background should be used for common setup for all scenarios, it should not also carry some test and verification clauses. so this "And" and "Then" should not be part of this setup, They should be a separate Scenario.